### PR TITLE
EZP-30915: [platform.sh]Fix for node version

### PR DESCRIPTION
**Pull request by damianz5**
[platform.sh]Fix for node version

https://jira.ez.no/browse/EZP-30915

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -92,6 +92,13 @@ hooks:
     build: |
         set -e
 
+        unset NPM_CONFIG_PREFIX
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | dash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+        nvm current
+        nvm install 12.6.0
```
Comment (SylvainGuittard) [inline .platform.app.yaml]: Second review. We recommend `Node.js 10.15.3 LTS` 
https://doc.ezplatform.com/en/latest/getting_started/requirements/#recommended-setups

File: .environment [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This is necessary for nvm to work.
```
Comment (andrerom) [inline .environment]: Anyway this can be placed in p.sh config/directory? Given it is p.sh specific.

**Comment by damianz5** [comment]
Updated - only necessary changes have been made

**Comment by andrerom** [comment]
@SylvainGuittard He moved it to build / deploy.

@damianz5 With the updated webpack encore, does it still now support the newer node version which is used by default? (its v12 right?)

**Comment by damianz5** [comment]
default node is 'node -v v6.17.1'.
we can omit .environment because its node/yarn is needed only build/deploy stage - no need to set the version for app itself

**Comment by damianz5** [comment]
ping @andrerom final review please, currently it contains only changes from https://docs.platform.sh/languages/nodejs/nvm.html and version bump

**Pull request merged**